docs/resources/aws_security_group.md.erb

```diff
@@ -121,6 +163,30 @@ A String reflecting the name that was given to the SG at creation time.
       its('group_name') { should cmp 'my_group' }
     end
 
+### inbound\_rules
+
+A list of the rules that the security group applies to incoming network traffic. This is a low-level property that is used by the [`allow_in`](#allow_in) and [`open_to_the_world`](#open_to_the_world) matchers; see them for detailed examples. `inbound_rules` is provided here for those wishing to use Ruby code to inspect the rules directly, instead of using higher-level matchers.
+
+Order is very important in these rules, as the sequentially first rule to match will be applied to network traffic. By default, AWS includes a reject-all rule as the last inbound rule.  This implicit rule will not appear in the inbound_rules list.
+
+If the security group could not be found (that is, `exists` is false), `inbound_rules` will return an empty list.
+
+    describe aws_security_group(group_name: linux_servers) do
```

Comment:
The sentence above is talking about a security group that cannot be found. This example is describing a look at the first rule. It may be useful to include an example that is tied to the sentence above.

The `linux_servers` variable is not defined and could create confusion.